src/MeshCore/Mesh.cpp

```diff
@@ -1 +1,159 @@
-#include "Mesh.h"
\ No newline at end of file
+#include "Mesh.h"
+
+MeshCore::Mesh::Mesh(){
+    _elementDataMap = { {ElementType::TETRA, {4, VTK_TETRA}},
+                        {ElementType::TRIANGLE, {3, VTK_TRIANGLE}},
+                        {ElementType::LINE, {2, VTK_LINE}}  };
+    _polyData = vtkSmartPointer<vtkPolyData>::New();
+    _vtkPoints = vtkSmartPointer<vtkPoints>::New();
+    _meshActor = vtkSmartPointer<vtkActor>::New();
+    gmsh::option::setNumber("Mesh.MeshSizeMin", 0.1);
+    gmsh::option::setNumber("Mesh.MeshSizeMax", 1);
+}
+
+void MeshCore::Mesh::addSizing(std::vector<int> verticesTags, double size){
+    MeshSizing meshSizing(verticesTags, size);
+    _meshSizings.push_back(meshSizing);
+};
+
+void MeshCore::Mesh::generateSurfaceMesh(){
+    for(const auto& meshSizing : _meshSizings){
+        gmsh::model::mesh::setSize(meshSizing.nodeTags,
+                                   meshSizing.elementSize);
+    }
+    gmsh::model::mesh::generate(2);
+
+    vtkSmartPointer<vtkCellArray> lineArray = getVtkCellArray(ElementType::LINE);
+    vtkSmartPointer<vtkCellArray> triangeArray = getVtkCellArray(ElementType::TRIANGLE);
+    vtkSmartPointer<vtkPoints> points = getVtkPoints();
+
+    _polyData->SetPoints(points);
+    _polyData->SetLines(lineArray);
+    _polyData->SetPolys(triangeArray);
+    this->_meshActor = createMeshActor(_polyData);
```

Comment:
In some places, as you refer to private attributes of a class you use this->_attribute, and in other places _attribute. Is this intentional?